apps/reaper/config/test.exs

```diff
@@ -1,7 +1,7 @@
 use Mix.Config
 
 config :logger,
-  level: :info
+  level: :warn
```

Comment:
Was this accidentally included?

Reply:
Yes.  Unfortunately, reaper's test logging was very noisy and this bit of configuration was overriding the configs for Andi.  So even though Andi had warn level logging for unit tests, the alphabet made it so that Reapers info logging won every time.  😦 

Unit test logging is much quieter now.  😄